pkg/cmd/roachtest/tests/mixed_version_job_compatibility_in_declarative_schema_changer.go

```diff
@@ -127,9 +123,14 @@ func executeSupportedDDLs(
 		`ALTER TABLE testdb.testsc.t3 VALIDATE CONSTRAINT check_positive_not_valid`,
 	}
 
+	// DDLs supported in v23_2.
+	v232DDLs := []string{
+		`CREATE SCHEMA testdb.foo`,
```

Comment:
correct me if I'm wrong: i tried checking the `process.go` dsc code + docs - seems like this is the only DSC thing shipped in 23.2

Reply:
I think there should have been no new schema changes in 23.2.

- CREATE SCHEMA was enabled in https://github.com/cockroachdb/cockroach/pull/120091
- CREATE SEQUENCE in https://github.com/cockroachdb/cockroach/pull/117793
- multiple ALTERs in one statement were in https://github.com/cockroachdb/cockroach/pull/119269

Reply:
oh right - create schema was version gated by 23.2, but it was not flipped "on" until faizan's PR